src/zb_links/db/alter_tables.py

```diff
@@ -0,0 +1,43 @@
+from flask import Blueprint
```

Comment:
This seems to be a hack. It is very complicated without solving an actual problem. The underlying design problem is that you should not manually assign ids in auto increment columns. I would suggest you require all the ids in the test code. We can expect that postgres does a decent job in managing this value.

Reply:
in the latest version id's are not assigned.  the fcn 'get_max_external_id' is just used to intialize the sequence for autoincrement, not to set an id.

Reply:
This should not be required. The sqlalchemy library takes care of it.